src/app/components/Modal/index.tsx

```diff
@@ -40,9 +38,6 @@ const Modal = ({
             </FormattedMessage>
             {children}
           </Styled.Body>
```

Comment:
i like adding the footer as a child of the modal, but putting it inside the body is going to make styling the body annoying. it would be good to also pull out the body of the modal into its own child, like this:

```
<Modal onModalClose={deny} heading='i18n:discard:heading'>
  <Body heading='i18n:discard:body'>this is the body</Body>
  <Footer>cool footer</Footer>
</Modal>;
```